### PR TITLE
WDP181202-11

### PR DESCRIPTION
**Pull request by Ma-Der**
Problem: No RWD for search bar and "Home" menu.
Solution: Added RWD to search bar and "Home" menu.

### CODE REVIEW

File: src/sass/components/_header.scss [hunk 1]
```diff
@@ -125,6 +125,36 @@ header {
...
+        text-align: center;
+        color: $text-color;
+        cursor: pointer;
+        position: absolute;
+        bottom: 10%;
+        left: 70%;
+
+        span {
+          background-color: $header-bg;
+          border-radius: 1px 1px 1px 1px;
```
Comment (pawel-schmidt) [inline src/sass/components/_header.scss]: You may use shorthand property `border-radius: 1px;`.

File: src/sass/components/_header.scss [hunk 2]
```diff
@@ -224,3 +254,60 @@ header {
...
+
+  .menu {
+    justify-content: center;
+  }
+}
+
+@media (max-width: 767px) {
+
+  .nav-hamburger {
+    display: block !important;
```
Comment (pawel-schmidt) [inline src/sass/components/_header.scss]: Using `!important` is a very bad practice. Do you have any idea to accomplish this task in another way?

This comment also applies to `!important`s below.

File: src/sass/components/_header.scss [hunk 3]
```diff
@@ -224,3 +254,60 @@ header {
...
+        a {
+          border-top: 4px solid $primary !important;
+          justify-content: center;
+          align-items: center;
+        }
+      }
+    }
+  }
+
+  .menu  #input-hamburger:checked + ul {
```
Comment (pawel-schmidt) [inline src/sass/components/_header.scss]: You can nest `#input-hamburger:checked + ul` inside `.menu` selector above.

File: src/sass/components/_header.scss (hunk 2 above)
Comment (Ma-Der) [reply src/sass/components/_header.scss]: I just did that in other way, think the way that should be from the start.

**Comment by xkxd** [comment]
please resolve conflicts

**Comment by Ma-Der** [comment]
Done, conflicts resolved.

**Pull request merged**